ParkingZoneApp/Models/Reservation.cs

```diff
@@ -22,7 +22,10 @@ public class Reservation
         public virtual ParkingSlot ParkingSlot { get; set; }
 
         [Required]
-        public int UserId { get; set; }
+        public string VehicleNumber { get; set; }
+
+        [Required]
+        public string UserId { get; set; }
```

Comment:
public string UserId { get; set; }
does this userId belong to ApplicationUser if yes I guess it should be type of Guid not string

Reply:
![image](https://github.com/SamanKingDotNetDeveloper/Parking_Zone_Client/assets/100861214/22ca1147-2361-4705-b57b-a05b6e78bb57)
in Controller here to get userId Im using this method and it returns string if i do UserId property Guid the function is giving exception.